interview-questions.md

```diff
@@ -6,42 +6,43 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-Type your initials here to confirm you read the instructions:
+Type your initials here to confirm you read the instructions: ms
 
-INSTRUCTOR EXAMPLE: What is a conditional statement?
+INSTRUCTOR EXAMPLE: What is a conditional statement? 
+Your answer: It is a statement stating that if this set of things happen then this code needs to run.
 
-Your answer:
 
-Researched answer:
+Researched answer:A conditional statement in JavaScript is a programming construct that allows you to execute different blocks of code based on whether a specified condition evaluates to true or false.
 
-1. What is git? What is the difference between git and Github?
+1. What is git? What is the difference between git and Github? 
 
-Your answer:
 
-Researched answer:
+Your answer:"git" I a command. GitHub is a website platform to store code and projects.
+
+Researched answer:So, Git is a version control system for tracking changes in code. Git vs Github, Git is the tool, Github is the platform for hosting Git repositories and collaboration.
 
 2. What is the difference between an array's index and its value?
 
-Your answer:
+Your answer: Index of an array in the positioning. Value it the actual worth.
 
-Researched answer:
+Researched answer: An, array index is the position of an element in the array. Array value is the data stored at a specific index.
 
 3. Compare and contrast arrays and strings. Please include ways in which they are the same and ways in which they are different.
 
-Your answer:
+Your answer: Array is like a list in a certain order. A string is like a word, or characters together in quotations. They are the same because they are in a sequence. But arrays are in brackets and strings are in quotations. 
 
-Researched answer:
+Researched answer: Arrays are collections of elements accessed by index, versatile. Strings are a sequences of characters, immutable in many languages, specialized for text manipulation.
 
-4. STRETCH: What are the four pillars of object oriented programming?
+4. STRETCH: What are the four pillars of object oriented programming? I don't know.
 
-Researched answer:
+Researched answer:  Encapsulation, abstraction, inheritance, and polymorphism.
```

Comment:
Good work overall on these questions!  Think about giving examples or other information around the topics as further evidence of your understanding.  Remember, these are interview questions, so how can you best show your future employer your knowledge.